target/ppc/translate/fixedpoint-impl.c.inc

```diff
@@ -184,36 +200,7 @@ TRANS64(LDUX, do_ldst_X, true, false, MO_Q)
 TRANS64(PLD, do_ldst_PLS_D, false, false, MO_Q)
 
 /* Load Quadword */
-static bool trans_LQ(DisasContext *ctx, arg_D *a)
-{
-    REQUIRE_INSNS_FLAGS(ctx, 64BX);
-
-    if (unlikely(a->rt & 1) || a->ra == a->rt) {
-        gen_inval_exception(ctx, POWERPC_EXCP_INVAL_INVAL);
-        return true;
-    }
-
-    if (!(ctx->insns_flags2 & PPC2_LSQ_ISA207)) {
-        if (ctx->pr) {
-            /* lq and stq were privileged prior to V. 2.07 */
-            gen_priv_exception(ctx, POWERPC_EXCP_PRIV_OPC);
-            return true;
-        }
-
-        if (ctx->le_mode) {
-            gen_align_no_le(ctx);
-            return true;
-        }
-    }
-
-    if (ctx->le_mode) {
-        return do_ldst_quad(ctx, a->ra, cpu_gpr[a->rt + 1], cpu_gpr[a->rt],
-                            tcg_constant_tl(a->si), false);
-    }
-    return do_ldst_quad(ctx, a->ra, cpu_gpr[a->rt], cpu_gpr[a->rt + 1],
-                        tcg_constant_tl(a->si), false);
-}
-
+TRANS64(LQ, do_ldst_quad, false, false);
```

Comment:
Are you sure we don't need 64BX?

I found this comment: https://github.com/qemu/qemu/commit/dfdd3e43620a6cd4f2be31da5a257c84a16fc000

Reply:
Fixup in https://github.com/PPC64/qemu/pull/14/commits/aa4560384f71f023265b42d49992bc59467dc085 added `REQUIRE_INSNS_FLAGS(ctx, 64BX);` in `do_ldst_quad`. I'm using `TRANS64` here just to allow the compiler to elide the call for qemu-ppc.

Reply:
Oops, I missed that.